Barotrauma/BarotraumaShared/SharedSource/Items/Components/Signal/Connection.cs

```diff
@@ -18,7 +18,15 @@ partial class Connection
         public readonly int MaxWires = 5;
 
         public readonly string Name;
-        public readonly LocalizedString DisplayName;
+
+        private readonly LocalizedString _displayName;
+        public LocalizedString DisplayName
+        {
+	        get => DisplayNameOverride ?? _displayName;
+	        private init => _displayName = value;
+        }
+
+        public LocalizedString DisplayNameOverride;
```

Comment:
```suggestion
        public LocalizedString? DisplayNameOverride;
```
I wanted to make it nullable but the nullable context is not enabled here.